app/renderer/lib/client-frameworks/robot.js

```diff
@@ -0,0 +1,102 @@
+import Framework from './framework';
+
+class RobotFramework extends Framework {
+
+  get language () {
+    //TODO: Make https://highlightjs.org/ use robot syntax
+    return "python";
+  }
+
+  get getCapsVariables() {
+    return Object.keys(this.caps).map((k) => {
+      return `${k}    ${this.getPythonVal(this.caps[k])}`;
+    }).join("\n");
+  }
+
+  getPythonVal (jsonVal) {
+    if (typeof jsonVal === 'boolean') {
+      return jsonVal ? "True" : "False";
+    }
+    return jsonVal;
+  }
+
+  wrapWithBoilerplate (code) {
+    return `# This sample code uses the Appium robot client
+# pip install robotframework-appiumlibrary
+# Then you can paste this into a file and simply run with robot
+#
+#  more keywords on: http://serhatbolsu.github.io/robotframework-appiumlibrary/AppiumLibrary.html
+
+*** Settings ***
+Library           AppiumLibrary
+
+*** Variables ***
+\$\{REMOTE_URL\}   ${this.serverUrl}
+${this.getCapsVariables}
+
+*** Test Cases ***
+Test case name
+${this.indent(this.getApplicationInitialization(),4)}
```

Comment:
spacing here and next line (should be `, 4`)

Reply:
Fixed on cefc256a875705a082bb2b1ac18451c5c3759103